BoxContentSDK/BoxContentSDK/Protocols/BOXContentCacheClientProtocol.h

```diff
@@ -292,6 +292,10 @@
 
 #pragma mark - Search
 
+- (void)cacheSearchRequest:(BOXSearchRequest *)request
```

Comment:
The answer is NO. Those params can be looked up from 'request' which has offset and limit as params.